files/setup-08-ingress.sh

```diff
@@ -21,7 +21,7 @@ kubectl --kubeconfig /root/.kube/config -n vmware create secret tls ${CERT_NAME}
 
 # Deploy Ingress Route
 
-if [ "${EVENT_PROCESSOR_TYPE}" == "AWS EventBridge" ]; then
+if [[ "${EVENT_PROCESSOR_TYPE}" == "AWS EventBridge" ]] || [[ "${EVENT_PROCESSOR_TYPE}" == "Knative" ]]; then
```

Comment:
Nit: to simplify this logic this could be flipped to check whether processor == `OPENFAAS` `or else` deploy the standard ingress